acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/node/PantheonNode.java

```diff
@@ -39,32 +47,37 @@
   private static final Logger LOG = LogManager.getLogger();
   private static final String PANTHEON_IMAGE = "pegasyseng/pantheon:latest";
 
+  /** Pantheon's dev.json has the hard fork at block 0. */
+  private static final BigInteger SPURIOUS_DRAGON_HARD_FORK_BLOCK = BigInteger.valueOf(1);
```

Comment:
shouldn't this be configurable especially if using custom genesis files

Reply:
🤷‍♀ It has to come from somewhere programatic, so moving that decision around doesn't help today. Perhaps tomorrow when there's a need for it YANGI